### DISCUSSION TITLE
Could I use  HEK293T-KO samples as unmodified (m6A) controls for data from SQK-RNA002?

### DISCUSSION THREAD

**Discussion by mantonglaw** [GoekeLab/xpore#121] [Q&A] [answered]
My data is from SQK-RNA002 , I notice that There is "Note that all HEK293T-KO samples can be used as unmodified (m6A) controls for any other data set generated with the same RNA kit (SQK-RNA001)" in your docs. Our samples don't have control groups, so I want to know if HEK293T-KO samples can be used as unmodified (m6A) controls  for our data. Thanks in advance!

**Comment by yuukiiwa** [comment, marked as answer]
Hi @mantonglaw,

Yes, you can use HEK293T-KO samples as unmodified (m6A) controls for data from SQK-RNA002. Replicates 2 and 3 of the HEK293T-KO samples were prepared using SQK-RNA002 while replicate 1 was prepared using SQK-RNA001. All three replicates can be used as unmodified controls even though one of the replicates used the SQK-RNA001 kit.
Thanks!

Best wishes,
Yuk Kei

**Comment by mantonglaw** [comment]
Thank you, it means a lot to me !

**Reply by yuukiiwa** [reply]
No problem! Glad it helps!

**Comment by mantonglaw** [comment]
@yuukiiwa 
Excuse me, I have another question. Should I use parameter "--rna"  when I run the process of Nanopolish/f5c eventalign?
Thank you very much!

**Reply by yuukiiwa** [reply]
Hi @mantonglaw,

You can run `nanopolish eventalign` according to [xpore's documentation](https://xpore.readthedocs.io/en/latest/preparation.html):
```
nanopolish index -d <PATH/TO/FAST5_DIR> <PATH/TO/FASTQ_FILE>

nanopolish eventalign --reads <PATH/TO/FASTQ_FILE> \
--bam <PATH/TO/BAM_FILE> \
--genome <PATH/TO/FASTA_FILE \
--signal-index \
--scale-events \
--summary <PATH/TO/summary.txt> \
--threads 32 > <PATH/TO/eventalign.txt>
```
Thanks!

Best wishes,
Yuk Kei

**Comment by huawen-poppy** [comment]
Hello, @yuukiiwa  . I have two samples from non-model species using SQK-RNA002. These two samples are from two conditions, one is under stress, and the other one is not. I don't have the unmodified controls. I am wondering, in my case, can I use the HEK293T-KO samples as unmodified (m6A) controls? Or I could directly set one of the samples as control to find the differential modification sites?

Thanks in advance for your help!

Best regards,
Huawen

**Reply by yuukiiwa** [reply]
Hi @huawen-poppy,

Very sorry for the delayed reply! I just saw this!

I would suggest not using the HEK293T-KO samples as an unmodified control for m6A as the species differ from human, where METTL3 may not be an m6A writer in your species and also potentially there will be large difference between your species and human, where the differential modification detect may not be detecting m6A.

Thanks!

Best wishes,
Yuk Kei
